src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -973,12 +973,6 @@ internal override void AddSynthesizedReturnTypeAttributes(PEModuleBuilder module
                 AddSynthesizedAttribute(ref attributes,
                     DeclaringCompilation.SynthesizeTupleNamesAttribute(type.TypeSymbol));
             }
-
-            if (type.ContainsNullableReferenceTypes())
```

Comment:
>if (type.ContainsNullableReferenceTypes()) [](start = 12, length = 42)

Is there a reason to leave dynamic and tuple names handling here rather than moving them to the base as well?

Reply:
Moved.